portfolio/src/main/webapp/index.html

```diff
@@ -412,25 +414,51 @@ <h2 class="text-uppercase">George Floyd</h2>
                 <div class="modal-body">
                   <!-- Project Details Go Here-->
                   <h2 class="text-uppercase">Ahmaud Arbery</h2>
-                  <p class="item-intro text-muted">Lorem ipsum dolor sit amet consectetur.</p>
+                  <p class="item-intro text-muted">I Run with Maud</p>
                   <img
                     class="img-fluid d-block mx-auto"
                     src="assets/img/martyrs/AhmaudArbery.jpg"
                     alt="Ahmaud Arbery"
                   />
-                  <p>
-                    Use this area to describe your project. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-                    Est blanditiis dolorem culpa incidunt minus dignissimos deserunt repellat aperiam quasi sunt officia
-                    expedita beatae cupiditate, maiores repudiandae, nostrum, reiciendis facere nemo!
-                  </p>
+                  <h4>Justice for Ahmaud Arbery</h4>
                   <ul class="list-inline">
-                    <li>Date: January 2020</li>
-                    <li>Client: Explore</li>
-                    <li>Category: Graphic Design</li>
+                    <li><b>Age:</b> 25</li>
+                    <li><b>Location of Death:</b> Brunswick, Georgia</li>
                   </ul>
+                  <p>
+                    On February 23, 2020, Ahmaud Arbery went for a jog in Satilla Shores when he ran past the front yard
+                    of Gregory McMichael. Gregory believed that Mr. Arbery looked similar to a man responsible for
+                    several break-ins in the area, so he called his son, Travis McMichael. Both Gregory and Travis
+                    grabbed a .357 Magnum handgun and shotgun, and chased Mr. Arbery in a pickup truck. The McMichael's
+                    attempted to cut Mr. Arbery off, but were unsuccessful as he began to <b>run for his life</b>. As he
+                    ran, he passed the truck of William Bryan Jr., the McMichael's neighbor, who recorded and eventually
+                    joined the chase. Mr. Arbery ran until he was trapped between the two trucks, and Travis McMichael
+                    got out of his truck with the shotgun. Mr. Arbery fought for his life when he was
+                    <b>shot 3 times</b> by Travis, followed by the yelling of a racial slur. Mr. Arbery stumbled and
+                    fell in the middle of the street, taking his last breath. All 3 men have since been arrested 2
+                    months later and indicted on 9 counts, including felony murder.
+                  </p>
+                  <hr />
+                  <h4>How to Help</h4>
+                  <p>
+                    <b>Donate Here: </b
+                    ><a class="interactive-pics" href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a
```

Comment:
This may not need to be changed, but this format seems  off to me. Like you can close the tag on line 444 and same thing on 445.

Reply:
I agree but it automatically formats html that way after running make pretty